examples/amd64/main.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "< 6.0"
+      version = ">= 6.0.0"
```

```suggestion
      version = "< 7.0.0"
```
Just so when a newer version inevitably releases it wont bump automatically.